src/tool/subcommands/api_cmd/api_compare_tests.rs

```diff
@@ -1742,6 +1736,41 @@ fn eth_tests_with_tipset<DB: Blockstore>(store: &Arc<DB>, shared_tipset: &Tipset
     tests
 }
 
+fn read_state_api_tests(tipset: &Tipset) -> anyhow::Result<Vec<RpcTest>> {
+    let tests = vec![
+        RpcTest::identity(StateReadState::request((
+            Address::SYSTEM_ACTOR,
+            tipset.key().into(),
+        ))?),
+        RpcTest::identity(StateReadState::request((
+            Address::SYSTEM_ACTOR,
+            Default::default(),
+        ))?),
+        RpcTest::identity(StateReadState::request((
+            Address::CRON_ACTOR,
+            tipset.key().into(),
+        ))?),
+        RpcTest::identity(StateReadState::request((
+            Address::MARKET_ACTOR,
+            tipset.key().into(),
+        ))?),
+        RpcTest::identity(StateReadState::request((
+            Address::new_id(1234), // account actor address `t01234`
+            tipset.key().into(),
+        ))?),
+        RpcTest::identity(StateReadState::request((
+            Address::new_id(78216), // miner actor address `f078216`
```

Comment:
```suggestion
            Address::new_id(78216), // miner actor address `t078216`
```

Also, might be worth linking to a relevant page on a block explorer.